### PR TITLE
🧼 URN code cleanup

### PR DESCRIPTION
**Pull request by rowanseymour**
Now that mailroom is handling URN validation for contact creation and updating, there is a lot of URN related code only being used by tests.

### CODE REVIEW

File: temba/channels/types/twitter/tests.py [hunk 1]
```diff
@@ -143,82 +141,6 @@ def test_release(self, mock_delete_webhook):
         self.channel.release(self.admin)
         mock_delete_webhook.assert_called_once_with("beta", "1234567")
 
-    @patch("twython.Twython.lookup_user")
-    @mock_mailroom
-    def test_resolve(self, mr_mocks, mock_lookup_user):
```
Comment (rowanseymour) [inline temba/channels/types/twitter/tests.py:148]: This channel type has been beta only for years as a way to hide it and the API is out of date.. and it's not even Twitter anymore.. so probably gonna nix the entire channel type next.

File: temba/tickets/tests.py [hunk 2]
```diff
@@ -873,7 +875,7 @@ def test_export(self):
                     ticket3.contact.uuid,
                     "Roy Kent",
                     "tel",
-                    "+1234567890",
+                    "+12345678900",
```
Comment (rowanseymour) [inline temba/tickets/tests.py:878]: added a zero to make this an actual "valid" US number

File: temba/utils/management/commands/mailroom_db.py [hunk 3]
```diff
@@ -436,9 +436,7 @@ def create_group_contacts(self, spec, org, user):
                 contacts = []
                 for i in range(size):
                     urn = f"tel:+250788{i:06}"
-                    contact = ContactURN.lookup(org, urn)
```
Comment (rowanseymour) [inline temba/utils/management/commands/mailroom_db.py:439]: wasn't used and this method doesn't return a contact

File: temba/api/v2/tests.py [hunk 4]
```diff
@@ -394,7 +394,7 @@ class EndpointsTest(APITest):
     def setUp(self):
         super().setUp()
 
-        self.joe = self.create_contact("Joe Blow", phone="0788123123")
+        self.joe = self.create_contact("Joe Blow", phone="+250788123123")
```
Comment (rowanseymour) [inline temba/api/v2/tests.py:397]: making these valid without needing normalization

**Review by norkans7 — APPROVED** [review]

**Pull request merged**